src/main/java/org/apache/maven/shared/utils/logging/MessageBuilder.java

```diff
@@ -21,6 +21,8 @@
 
 /**
  * Message builder that supports configurable styling.
+ * An instance of this interface can be retrieved with {@link MessageUtils#buffer()}.
+ * After the message has been build with any of the append methods its content can be retrieved with {@link #build()}.
```

Comment:
_build, built, **built**_

Reply:
Thanks, replaced by "constructed" in https://github.com/apache/maven-shared-utils/pull/96/commits/35eb1e3a0dd029fb872b153fc04c847089b07377 as "build/built" is kind of overloaded here.